enterprise/backend/src/metabase_enterprise/workspaces/context.clj

```diff
@@ -87,3 +87,15 @@
        (-> (select-keys src-output-id->dst-output deps-ids)
            (update-keys src-id->schema+table)
            (update-vals (juxt :schema :name))))))
+
+(defn transform-node->py-tables-remaps
```

Comment:
I believe there are transforms interfaces for accessing this, it would be nice to respect their encapsulation.